### PR TITLE
Add gender to edit profile page closes #305

### LINKED ISSUE #305: Add gender to edit profile page
Currently we don't have the gender field displayed on the user profile page, so now its only possible to add that info when registering 😜 

Thanks @bex1 for pointing it out 👍 https://github.com/justarrived/just-match-web/issues/292#issuecomment-279989603 (next time convert it to an issue directly) :)

### PR DESCRIPTION
**Pull request by Ekman1991**

### CODE REVIEW

**Comment by buren** [comment]
I like it 👍 Perhaps we should extract this to its own component `<user-gender-select>` or sth like that while we're at it.

**Comment by Ekman1991** [comment]
Is it not better to keep the implementation of `<user-gender-select>` in the existing pull request (https://github.com/justarrived/just-match-web/pull/370) ?

File: src/app/components/forms/user-details-form/user-details-form.component.ts [hunk 1]
```diff
@@ -16,6 +17,7 @@ import {Validators} from '@angular/forms';
 })
 export class UserDetailsFormComponent implements OnInit {
   @Input() user: User;
+  private genders: Promise<UserGender[]>;
```
Comment (bex1) [inline src/app/components/forms/user-details-form/user-details-form.component.ts]: SHould be public

File: src/app/components/forms/user-details-form/user-details-form.component.ts [hunk 2]
```diff
@@ -33,6 +35,11 @@ export class UserDetailsFormComponent implements OnInit {
   ngOnInit() {
     this.initSettingsForm();
     this.initPasswordForm();
+    this.loadData();
+  }
+
+  loadData() {
```
Comment (bex1) [inline src/app/components/forms/user-details-form/user-details-form.component.ts]: Extend TranslationsListener. See conventions please.
Its not enough to just add loadData function.

**Review by bex1 — CHANGES_REQUESTED** [review]
Good 👍 Just fix my two comments then merge

**Review by bex1 — APPROVED** [review]

**Comment by bex1** [comment]
This will not be merged. Solved in #386 instead

**Pull request closed without merging**